### PR TITLE
clean up information about porting Android apps

### PR DESCRIPTION
**Pull request by ru-fu**
This information is how-to rather than explanation.

Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: howto/port/configure-watchdog.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+The [watchdog](https://discourse.ubuntu.com/t/application-manifest/24197#watchdog) monitors the app installed by the boot package. By default, it terminates the container if the app fails or is moved to the background.
+
```
Comment (adglkh) [inline howto/port/configure-watchdog.md:2]: if the app fails ....
->
if the app crashes ...

File: howto/port/landing.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+When porting an Android app to Anbox Cloud (usually in the form of an APK), there are a few issues that might cause your app to not function properly:
+
+* Missing dependencies, most importantly to Google Play services. Google Play services are not supported by Anbox Cloud, and apps that require Google Play services can therefore not be ported to Anbox Cloud.
+* Missing runtime permissions. See [Grant runtime permissions](tbd) for instructions on how to grant the required permissions.
```
Comment (adglkh) [inline howto/port/landing.md:4]: Those `tbd`s are placeholders and will be replaced with the real URL when we publish these?

File: howto/port/obb-files.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+Android [has a 100 MB limit on apps](https://developer.android.com/google/play/expansion-files.html). Applications for apps that are under that limit can be created using the [regular method](https://discourse.ubuntu.com/t/create-an-application/24198).
+
+Larger Android apps must be split up into the main part as an APK file and the expansion part as an OBB file:
+
+- APK: Contains the executables and native libraries (`*.so` files), plugins, basic assets and data required by the application to load for the first time.
+- OBB: Contains the remaining assets (high-fidelity graphics, media files or other large resource files) for the full user experience.
+
+Anbox Cloud supports creating application for such apps, but it requires additional steps.
```
Comment (adglkh) [inline howto/port/obb-files.md:8]: ... creating application ...
... creating applications ...

**Review by adglkh — CHANGES_REQUESTED** [review]
Just a couple of nitpicks.

File: howto/port/landing.md (hunk 2 above)
Comment (ru-fu) [reply howto/port/landing.md:4]: Yes, exactly. We don't know the URL before publishing, unfortunately. :(

**Comment by ru-fu** [comment]
> Just a couple of nitpicks.

Thanks a lot!
Updated. :)

File: howto/port/landing.md (hunk 2 above)
Comment (adglkh) [reply howto/port/landing.md:4]: I see.

**Review by adglkh — APPROVED** [review]
LGTM

**Pull request merged**